flow/Makefile

```diff
@@ -565,11 +569,13 @@ define do-step
 $(if $(5),$(5),$(RESULTS_DIR))/$(1)$(if $(4),$(4),.odb): $(2)
 	$$(UNSET_AND_MAKE) do-$(1)
 
-.PHONY: do-$(1)
+.PHONY: do-$(1) copyright.txt
 do-$(1):
```

Comment:
This doesn't introduce any dependency.  Are you intending for copyright.txt to be automatically generated or just a user target that will never be used?

Reply:
This just.makes sure that the copyright is printed *once* to the console.

Reply:
I don't think it does - you just declared it a phony not a dependency

Reply:
Yes, I am off by one line, my eyes are not what they used to be :-)

Reply:
fixed